cmd/explaintest/r/topn_push_down.result

```diff
@@ -168,22 +168,22 @@ ORDER BY te.expect_time asc
 LIMIT 0, 5;
 id	count	task	operator info
 Projection_13	0.00	root	test.te.expect_time
-└─Limit_19	0.00	root	offset:0, count:5
-  └─IndexJoin_92	0.00	root	left outer join, inner:IndexReader_91, outer key:test.tr.id, inner key:test.p.relate_id
-    ├─TopN_95	0.00	root	test.te.expect_time:asc, offset:0, count:5
-    │ └─IndexJoin_36	0.00	root	inner join, inner:IndexLookUp_35, outer key:test.tr.id, inner key:test.te.trade_id
-    │   ├─IndexLookUp_74	0.00	root	
-    │   │ ├─Selection_72	0.00	cop	eq(test.tr.business_type, 18), eq(test.tr.trade_type, 1)
-    │   │ │ └─IndexScan_70	10.00	cop	table:tr, index:shop_identy, trade_status, business_type, trade_pay_status, trade_type, delivery_type, source, biz_date, range:[810094178,810094178], keep order:false, stats:pseudo
-    │   │ └─Selection_73	0.00	cop	eq(test.tr.brand_identy, 32314), eq(test.tr.domain_type, 2)
-    │   │   └─TableScan_71	0.00	cop	table:tr, keep order:false, stats:pseudo
-    │   └─IndexLookUp_35	0.25	root	
-    │     ├─IndexScan_32	10.00	cop	table:te, index:trade_id, range: decided by [eq(test.te.trade_id, test.tr.id)], keep order:false, stats:pseudo
-    │     └─Selection_34	0.25	cop	ge(test.te.expect_time, 2018-04-23 00:00:00.000000), le(test.te.expect_time, 2018-04-23 23:59:59.000000)
-    │       └─TableScan_33	10.00	cop	table:te, keep order:false, stats:pseudo
-    └─IndexReader_91	9.99	root	index:Selection_90
-      └─Selection_90	9.99	cop	not(isnull(test.p.relate_id))
-        └─IndexScan_89	10.00	cop	table:p, index:relate_id, range: decided by [eq(test.p.relate_id, test.tr.id)], keep order:false, stats:pseudo
+└─TopN_16	0.00	root	test.te.expect_time:asc, offset:0, count:5
```

Comment:
It's wried, it should be a `Limit` operator?

Reply:
It's the result of `rule_topn_push_down`.